en/patterns/menu.md

```diff
@@ -6,16 +6,21 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
 
 # Menu
 
-Use the Menu Symbols as presets for building navigation bar for application or webpage that contains elements such as logo, menu items, search input, various actions, account settings.
-
+Use the Menu Symbols as presets for building navigation bars for applications or webpages containing elements such as a logo, menu items, a search, various actions, and account settings among others.
 
 <img class="responsive-img" src="../images/menu.png" srcset="../images/menu@2x.png 2x" />
 
+## Orientation
+
+The Menu presets are grouped according to their orientation into Horizontal and Vertical.
+
+<img class="responsive-img" src="../images/menu_horizontal.png" srcset="../images/menu_horizontal@2x.png 2x" />
+<img class="responsive-img" src="../images/menu_vertical.png" srcset="../images/menu_vertical@2x.png 2x" />
```

Comment:
@sdimchevski create and add images that show some horizontal and vertical menu layouts.

Reply:
I did. Have a look